src/hardware/mixer.cpp

```diff
@@ -2354,10 +2354,10 @@ class MIXER final : public Program {
 		                        const std::string &chorus) {
 			WriteOut(column_layout.c_str(),
 			         name.c_str(),
-			         volume.left * 100.0f,
-			         volume.right * 100.0f,
-			         gain_to_decibel(volume.left),
-			         gain_to_decibel(volume.right),
+			         static_cast<double>(volume.left * 100.0f),
+			         static_cast<double>(volume.right * 100.0f),
```

Comment:
`gain_to_percentage` to improve readability?

Reply:
Also renamed `percent_to_ratio` to `percentage_to_gain` to follow the pattern.